src/context/Theme.tsx

```diff
@@ -156,6 +159,11 @@ export const tableAlternateRowsSx: SxProps<Theme> = {
     },
 };
 
+// TODO (optimization): Replace instances of outlineSx with slateOutline.
+export const slateOutline = {
+    200: `1px solid ${slate[200]}`,
+};
+
```

Comment:
This makes no sense to me. We are duplicating the `200` here for honestly very little code de-duplication of the `1px solid`. 

If we want to share these kind of properties this should be a function that consumes arguments to generate this so it works for any predefined color and shade.

Reply:
Will remove